### PR TITLE
Possible fix for the broken link for cache

### PR DESCRIPTION
**Pull request by chasen**

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1489,6 +1489,7 @@ throw
 Throw PHP errors as ``\ErrorException`` instances. The parameter
 ``debug.error_handler.throw_at`` controls the threshold.
 
+.. _reference-cache:
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: I believe you have to add a blank line after the label.

**Comment by xabbuh** [comment]
@chasen Thanks for noticing this and making this fix. I think adding the missing blank line will fully solve the issue.

File: reference/configuration/framework.rst (hunk 1 above)
Comment (chasen) [inline reference/configuration/framework.rst]: Aw yes, sorry i missed that.

**Comment by xabbuh** [comment]
Good catch, thanks @chasen.

**Pull request merged**